app/javascript/youtube.js

```diff
@@ -1,6 +1,7 @@
 const async = require('async')
 
 window.onYouTubeIframeAPIReady = () => {
+  setupYouTubeVideos()
   return true
 }
```

Comment:
Are we able to collapse `onYouTubeIframeAPIReady` and `setupYouTubeVideos` into a single function?

Reply:
I don't see why not.